agent/consul/server.go

```diff
@@ -872,12 +889,19 @@ func (s *Server) KeyManagerLAN() *serf.KeyManager {
 
 // KeyManagerWAN returns the WAN Serf keyring manager
 func (s *Server) KeyManagerWAN() *serf.KeyManager {
+	if s.serfWAN == nil {
+		return nil
+	}
```

Comment:
For the record I verified all call paths to this to check none will panic on a nil. It's only called from an internal state machine routine when working with WAN keyrings which should be disabled by code in this PR.

https://github.com/hashicorp/consul/blob/17681f04f96e1934a73f96ba6864f916c56e08fc/agent/consul/internal_endpoint.go#L143-L151

called from 

https://github.com/hashicorp/consul/blob/17681f04f96e1934a73f96ba6864f916c56e08fc/agent/consul/internal_endpoint.go#L131-L136

which is disabled above.